content/context-resource/index.md

```diff
@@ -0,0 +1,320 @@
+            , endDate : Timestamp
+            , replacement : Maybe Url
+            , reason : Text
+            }
+```
+
+|Name|Description|
+|-|-|
+|Start date| Date when the register started.|
+|End date| Date when the register was retired.|
+|Replacement| A URL to a register that replaces the current one.|
+|Reason| A human readable explanation of why the register is no longer active.|
+
+
+#### Title
+
+* Type: Optional String
+
+The human readable name of the register.
+
+
+### HTTP resource
+
+***
+#### Endpoint
+
+```
+GET /context
+```
+
+#### Parameters
+
+|Name|Type|Description|
+|-|-|-|
+|`total-entries` | Optional `Integer`| The log size to compute the snapshot from. Note this only applies when the metadata can be versioned.|
+
+
+#### Response attributes
+
+|Name|Type|
+|-|-|
+|`id`| Name |
+|`copyright`| String |
+|`custodian`| Optional String |
+|`description`| Optional String |
+|`hashing-algorithm`| HashingAlgorithm |
+|`licence`| String |
+|`root-hash`| Hash |
+|`schema`| List Attribute |
+|`stats`| Stats |
+|`status`| Status |
+|`title`| Optional String |
+
+#### Hashing algorithm attributes
+
+|Name|Type|
+|-|-|
+|`digest-length`| Integer |
+|`function-type`| Integer |
+|`id`| Name |
+
+See also: [Multihash table function type
+identifiers](https://github.com/multiformats/multihash/blob/master/hashtable.csv).
+
+#### Attribute attributes
+
+|Name|Type|
+|-|-|
+|`id`| Name |
+|`datatype`| Datatype |
+|`cardinality`| Cardinality |
+|`title`| Optional String |
+|`description`| Optional String |
+
+#### Stats attributes
+
+|Name|Type|
+|-|-|
+|`total-entries`| Integer |
+|`total-items`| Integer |
+|`total-records`| Integer |
+
+#### Status attributes
+
+|Name|Type|
+|-|-|
+|`start-date`| Datetime |
+|`end-date`| Optional Datetime |
+|`replacement`| Optional Url |
+|`reason`| Optional String |
+
+***
+
+***
+**EXAMPLE:**
+
+
+```http
+GET /summary HTTP/1.1
+Accept: application/json
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "id": "multihash",
+  "title": "The Multihash register",
+  "description": "List of multihash codes.",
+  "custodian": "IPFS team",
+  "hashing-algorithm": {
+    "id": "sha2-256",
+    "function-type": 18,
+    "digest-length": 32
+  },
+  "stats": {
+    "total-entries": 0,
+    "total-items": 0,
+    "total-records": 0,
+  },
+  "copyright": "Copyright (c) 2016 Protocol Labs Inc.",
+  "licence": "MIT",
+  "root-hash": "1220e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+  "status": { "start-date": "2018-12" },
+  "schema": [
+    {"id": "name", "datatype": "name", "cardinality": "1"},
+    {"id": "function-type", "datatype": "integer", "cardinality": "1"},
+    {"id": "digest-length", "datatype": "integer", "cardinality": "1"}
+  ]
+}
+```
+
+***
+
+### Considerations
+
+The context resource makes the register resource obsolete. Given that users
+depend on it the register resource will be available for as long as necessary.
```

Comment:
How long is that? This shouldn't need to apply to new registers, and it would be great if we didn't have to support it in the reference implementation forever as well.

Reply:
I agree, I think this will be answered next quarter when we define the version strategy, but I'm open to suggestions